scripts/monitoring/cron-send-user-running-pod-count.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I suggest you change this to this:
```
#!/bin/bash -e
```

That way any errors will exit the script.

Then, ops-runner can alert us to the failure. Otherwise, you may exit with an exit code of 0, even though the script failed.